WooCommerce/Classes/POS/Presentation/Card Present Payments/Reader Messages/PointOfSaleTransactionSuccessViewModel.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-struct PointOfSaleCardPresentPaymentSuccessMessageViewModel: Equatable {
+struct PointOfSaleTransactionSuccessViewModel: Equatable {
```

Comment:
`PointOfSaleTransactionSuccessViewModel` ? `PointOfSalePaymentSuccessViewModel` maybe better ?  🤔

Reply:
`PointOfSalePaymentSuccessViewModel` sounds better to me 👍